src/SimpleIni/README.md

```diff
@@ -0,0 +1,9 @@
+# FRIENDLY NOTE
```

Comment:
maybe move `SimpleIni` to `lib/SimpleIni` to not pollute our source directory with it?

Reply:
sounds good